app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -25,5 +30,14 @@ class ProfileViewModelTest : ShouldSpec(
 
 			coVerify { authRepository.deleteAccount() }
 		}
+
+		should("profile return Profile data WHEN ProfileViewModelTest was initialized") {
+			runTest(StandardTestDispatcher()) {
+				val profile = Profile(null, "-", "-")
```

Comment:
Do we need a real object here ? we can just use mock as well here :) also please consider if runTest is really needed :)

Reply:
Mockk will be enough but what about runTest() without it and a specific dispatcher this test will fail.

Reply:
Done